lib/runner.rb

```diff
@@ -0,0 +1,4 @@
+require_relative "tictactoe"
+
+new_game = Tic_Tac_Toe.new
+new_game.show_empty_board
```

Comment:
Let's move this call into TicTacToe.run_app() 
and change the this to 
```
new_game = Tic_Tac_Toe.new
new_game.run_app
```

Reply:
👍🏽